### PR TITLE
members page done

### PR DESCRIPTION
**Pull request by Ash-0803**
Members page with html and css is done!

### CODE REVIEW

**Comment by pulkit-30** [comment]
Great work @Ash-0803 🎉 
but we need to improve UI... WDYT @IshaanTyagi?

**Comment by Ash-0803** [comment]
Yeah. Ui isn't any good right now, but we did that keeping in mind that I, ansh and ankur mainly will be working on the Ui that I have provided in the Ui issue shortly. Until then, we can do some refinements in this Ui as well. Actually I have asked ansh to do so too.

File: index.html [hunk 1]
```diff
@@ -4,7 +4,146 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <!-- links to css and js files -->
+    <link rel="stylesheet" href="./styles.css" />
+
     <title>Document</title>
```
Comment (pulkit-30) [inline index.html:17]: Add title as `NSCC Team`

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:400,500,600,700&display=swap");
+* {
+    font-family: "Poppins", sans-serif;
```
Comment (pulkit-30) [inline styles.css:3]: Add font as a root variable and use the root variable instead reset-CSS font-family globally

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+}
+
+body {
+    color: white;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
```
Comment (pulkit-30) [inline styles.css:10]: Add `margin` `padding` `box-sizing` in reset-css instead of font-family

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    background-color: #090d16 ;
```
Comment (pulkit-30) [inline styles.css:11]: Again, declare colors in root variable and then use them

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    position: relative;
+}
+.core-team {
+    transform: scale(0.9);
+    display: inline-block;
+    margin-bottom: 0rem;
+}
+
+.core-container {
+    background: linear-gradient(to right, rgba(15, 32, 39,0.8), rgba(32, 58, 67,0.5), rgba(44, 83, 100,0.2));
```
Comment (pulkit-30) [inline styles.css:59]: also, use this as root-variable

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

**Comment by pulkit-30** [comment]
ref https://www.w3schools.com/css/css3_variables.asp for declaring root-variables in css

File: styles.css (hunk 3 above)
Comment (pulkit-30) [reply styles.css:18]: now we can remove these properties from the body

**Review by pulkit-30 — APPROVED** [review]

**Pull request merged**